File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> retarr;
        for(int i=0;i<nums.size();i++){
            for(int j=i+1;j<nums.size();j++){
                if(nums[i]+nums[j]==target){
                    retarr.push_back(i);
                    retarr.push_back(j);
                    return retarr;
                }
            }
        }
        return{};
    }
    
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dict={}
        rett=[]
        for i in range(0,len(nums)):
            val=nums[i]
            if (target-nums[i]) in dict:
                rett.append(i)
                rett.append(dict[target-nums[i]])
                break
            else :
                dict[nums[i]]=i
        return rett